tern/analyze/passthrough.py

```diff
@@ -89,18 +89,21 @@ def run_on_image(image_obj, command, is_sudo=False):
         return False
     # execute for each layer object
     for layer in image_obj.layers:
-        # set that we're analyzing at the file level
-        layer.files_analyzed = True
-        # get the actual command
-        full_cmd = get_filesystem_command(layer, command)
-        if not execute_external_command(layer, full_cmd, is_sudo):
-            logger.error(
-                "Error in executing given external command: %s", command)
-            return False
+        # run the command at each layer if we are redoing or if there
+        # is nothing in the analyzed output
+        if redo or not layer.analyzed_output:
+            # set that we're analyzing at the file level
```

Comment:
I thought you were going to change this to `if redo or not layer.files_analyzed` to be the same as the implementation in the scancode extension?